extra/utils/Validation.js

```diff
@@ -1,24 +1,11 @@
-let users = [
-    {
-        traineeEmail : 'shashank.baranawal@successive.tech',
-        reviewerEmail : 'naman.parashar@successive.tech',
-    },
-    {
-        traineeEmail : 'shashank.baranwal@gmail.com',
-        reviewerEmail : 'naman.parashar@gmail.com'
-    }
-]
-
-function validateEmail(email){
-    const regix = /\w+.\w+@successive.tech$/i;
-    return regix.test(email)
-}
-
+import {user} from '../constant'
+import {validateEmail} from './helpers'
 
 //console.log(validateEmail(users[0]["traineeEmail"]))
 let a =[];
 let b =[];
 function validateUsers(users){
```

Comment:
function is not exported

Reply:
i go through this file and I found it then export the function.